server/views/shared/referralDetails.njk

```diff
@@ -31,7 +31,7 @@
 
       <h2 class="govuk-heading-m">Service user's risk information</h2>
 
-      {% if presenter.riskPresenter.riskSummaryEnabled %}
+      {% if presenter.riskPresenter.riskSummaryEnabled and not presenter.riskPresenter.riskSummaryNotFound %}
```

Comment:
_nit_ "NotFound" seems to be used in negatives across the board. Would it be simpler to use `riskSummaryAvailable`?

Reply:
i count this useage in the negative to two useages without negation 😛